### ISSUE TITLE
'await' outside function

### ISSUE THREAD

**Issue by jordanehli** [akaszynski/keepa#80]
Hello,
Thank you for building out this very helpful package! So while using it I began to get the "This event loop is already running" error" so in order to fix this I took the recommendations from Github and used asyncio. Now whenever I run this I am getting the error "SyntaxError: 'await' outside function" and it appears before both instances where I use await. Do you have any suggestions for how I could fix this? I've attached my code snippet below. Thank you!

import keepa
import asyncio

mykey = 'REMOVEDFORSAFETY' 
api = await keepa.AsyncKeepa.create(mykey)

asins = ['0022841350', '0022841369']
products = await api.query(asins)

**Comment by samamorgan** [comment]
You can't call await outside of an async function. I'd suggest reading this walkthrough on python AsyncIO: https://realpython.com/async-io-python/#a-full-program-asynchronous-requests

**Comment by chocobn69** [comment]
@jordanehli I guess your not running python with standard cpython env right ?
If you are, you should use keepa like this: 

```python
import keepa
api = keepa.Keepa(accesskey)
products = api.query('ASIN')
```

If you are running in a jupyter notebook or something similar, I need to investigate to help you.

**Comment by akaszynski** [comment]
Should be fixed in `pyvista==1.3.0`.
